tripal_chado/src/TripalImporter/ChadoImporterBase.php

```diff
@@ -244,4 +244,33 @@ public function addAnalysis($form, &$form_state) {
 
     return $element;
   }
+
+  /**
+   * Performs tasks after the importer has completed.
+   *
+   * If the importer form specified one or more bundles to be published,
+   * these will be listed under the 'publish' key in the run arguments.
+   *
+   * @return void
+   *   No return value.
+   */
+  public function postRun() {
+    parent::PostRun();
+
+    // Only publish if a publish manager has been set by a child class.
+    if ($this->publish_manager) {
+      $arguments = $this->getArguments();
+      $bundles = $arguments['run_args']['publish'] ?? [];
+      if ($bundles) {
+        $publish_options = [
+          'chado_storage' => [
+            'schema_name' => $arguments['run_args']['schema_name'],
+            'bundles' => $bundles,
+          ],
+        ];
+        $this->publish_manager->publishBundles($publish_options);
+      }
+    }
```

Comment:
I suggest we switch this around a bit to ensure we don't sublit a publish job when no bundles are to be published. This will also help us if we decide to set the manager more globally.

```suggestion
    $arguments = $this->getArguments();
    $bundles = $arguments['run_args']['publish'] ?? [];
    if ($this->publish_manager and !empty($bundles)) {
      $publish_options = [
        'chado_storage' => [
          'schema_name' => $arguments['run_args']['schema_name'],
          'bundles' => $bundles,
        ],
      ];
      $this->publish_manager->publishBundles($publish_options);
    }
```